scripts/ExportScripts/export_profile_monolithic.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+#Note: project_path and engine_path are computed before this script runs, and can be accessed via o3de_context.project_path
+#for now this script assumes an engine-centric approach, and we assume to use the source built engine for the engine_path
+
+#to use this script, go to the engine root directory, and issue the following command:
+#.\scripts\o3de.bat export-project -pp C:\path\to\project -es C:\path\to\export_profile_monolithic.py -nmp C:\path\to\build\non_mono -mp C:\path\to\build\mono -out C:\path\to\output -ll INFO
+
+parser = argparse.ArgumentParser(prog="Profile Monolithic exporter for windows",
+                                 description="Exports a profile monolithic packaged build for the project")
+parser.add_argument('-nmp', '--non-mono-build-path', type=pathlib.Path, required=True)
+parser.add_argument('-mp', '--mono-build-path',type=pathlib.Path, required=True)
+parser.add_argument('-out', '--output-path', type=pathlib.Path, required=True)
+
+args = parser.parse_args(o3de_context.args)
+
+projects_path_cmake_option = f"-DLY_PROJECTS={str(o3de_context.project_path)}"
+
+project_json_data = manifest.get_project_json_data(project_path = o3de_context.project_path)
+project_name= project_json_data.get('project_name')
+
+
+#build non-monolithic artifacts
+process_command(['cmake', '-S', '.', '-B', args.non_mono_build_path, '-DLY_MONOLITHIC_GAME=0', projects_path_cmake_option], cwd= o3de_context.engine_path)
+
+process_command(['cmake', '--build', args.non_mono_build_path, '--target', 'Editor', '--target', 'AssetBundler', '--target', 'AssetProcessor', '--target', 'AssetProcessorBatch', '--config', 'profile'], cwd=o3de_context.engine_path)
+
+executables_dir = args.non_mono_build_path / 'bin' / 'profile'
+
+asset_processor_exe = executables_dir / 'AssetProcessorBatch.exe'
+
+asset_bundler_exe = executables_dir / 'AssetBundler.exe'
+
+process_command([asset_processor_exe, '--project-path', o3de_context.project_path], cwd=o3de_context.engine_path)
+
+
+#build monolithic artifacts
+process_command(['cmake', '-S', '.', '-B', args.mono_build_path, '-DLY_MONOLITHIC_GAME=1', '-DALLOW_SETTINGS_REGISTRY_DEVELOPMENT_OVERRIDES=0', projects_path_cmake_option], cwd = o3de_context.engine_path)
+process_command(['cmake', '--build',  args.mono_build_path, '--target', f'{project_name}.GameLauncher', '--target', f'{project_name}.ServerLauncher', '--target', f'{project_name}.UnifiedLauncher', '--config', 'profile'])
+
+#bundle content (WIP)
+# process_command([asset_bundler_exe, f'--project-path="{str(o3de_context.project_path)}"'])
+
+#copy artifacts to output directory
+
+pc_pak_path = args.output_path / 'Cache' / 'pc'
+
+os.makedirs(pc_pak_path, exist_ok=True)
+# os.makedirs(args.output_path / 'Gems' / 'AWSCore')
+process_command(['copy', str(o3de_context.project_path / 'AssetBundling' / 'Bundles' / '*.pak'), str(pc_pak_path)])
+process_command(['copy', str(args.mono_build_path / 'bin' / 'profile' / '*.*'), str(args.output_path)])
```

Comment:
The CMake install command should be used for this, as that will also fixup any `rpaths` on Linux and Mac needed to load any shared library(.so)/dynamic library(.dylib) in a standalone fashion.
Furthermore it fixes the file permissions as well, if some files need to be marked with the executable bit or read-only, etc...

It is also more portable as it doesn't assume that the CMAKE_RUNTIME_OUTPUT_DIRECTORY is always set to `<build-dir>/bin/<config>`
```suggestion
process_command(['cmake', '--install', f'{args.mono_build_path}', '--prefix', f'{args.output_path}', '--config', args.mono_config])
```